README.md

```diff
@@ -1 +1,12 @@
-# actions-collection
\ No newline at end of file
+# actions-collection
+
+## Description
+
+Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet)
+
+To run shell functions from collection.sh:
+
+```bash
+source collection.sh
+sh collection.sh docker_publish
```

Comment:
Can't you simply do `docker_publish` once you do `source`?

Reply:
I think so but I feel like if we don't specify the file: the memory might (shouldn't) have another function named docker_publish.

Reply:
but im okay with just ```docker_publish```

Reply:
The memory shouldn't be having that. If it has, then we are doing something wrong somewhere.

Reply:
Fixed.